sdk/python/kfp/_client.py

```diff
@@ -47,33 +47,22 @@ def __init__(self, host=None, client_id=None):
     """
 
     try:
-      import kfp_experiment
+      import kfp_server_api
```

Comment:
I considered those names, but I wanted to clearly differentiate the server/backend API from the DSL API.
I'm fine with changing the name later (it's OK since no one will use it directly).